packages/app/src/components/sidebar-metric/sidebar-metric.tsx

```diff
@@ -90,9 +90,10 @@ export function SidebarMetric<T extends { difference: unknown }>({
    * @TODO this should really be called sidebar_metric_description or something
    * as it's not a title at all.
    */
+  const text = siteText[localeTextKey];
   const title =
-    get(siteText, [localeTextKey, 'kpi_titel']) ||
-    get(siteText, [localeTextKey, 'titel_kpi']);
+    (typeof text === 'object' && 'kpi_titel' in text && text.kpi_titel) ||
+    (typeof text === 'object' && 'titel_kpi' in text && text.titel_kpi);
```

Comment:
Similar question here:

```suggestion
    text?.kpi_titel || text?.titel_kpi;
```

Reply:
That won't work actually, because `text` is a union of many types. Therefore the compiler will bark at you if you just use the `?` shortcut.